schemas/rules/operators/comparison/greater-than-or-equal.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://eq.ons.gov.uk/rules/operators/comparison/greater-than-or-equal.json",
+  "description": "Evaluates greater than or equal, takes two arguments",
+  "type": "object",
+  "properties": {
+    ">=": {
+      "type": "array",
+      "minItems": 2,
+      "maxItems": 2,
+      "items": {
+        "anyOf": [
+          {
+            "$ref": "https://eq.ons.gov.uk/rules/definitions.json#/number_value"
+          },
+          {
+            "$ref": "https://eq.ons.gov.uk/rules/definitions.json#/date_value"
+          },
```

Comment:
We should support `any_value` here as we would want this usable with metadata source etc?. Applies to other schemas where this is used also. I think there is only so much we can do in schemas, but we should allow the likes of metadata as a source, even if that could be of any type. I think at schema level as long as we validate it is one of the value sources we support, then it's fine.

Reply:
As these operators won't support arrays, I have used the `any_value_except_array` in https://github.com/ONSdigital/eq-questionnaire-validator/pull/96/commits/2c073ca22ce6e90ac03b8b4e943c6bf121e3a048